site/docs/0.0/assets/scss/_custom-docs.scss

```diff
@@ -109,6 +109,18 @@
   }
 }
 
+/* stylelint-disable */
+// >> Off-canvas Nav example
+@include media-breakpoint-down(md) {
+
+  .navbar-offcanvas {
```

Comment:
This rule isn't getting applied since since the classes are mismatched. Once we change the class name back to `.navbar-offcanvas` instead of `.navbar-off-canvas`, this will fix the gap that's appearing to the right of the mobile nav on the Bootstrap docs site.